src/functionalTest/java/uk/gov/hmcts/reform/draftstore/features/store/CreateDraftTest.java

```diff
@@ -0,0 +1,37 @@
+package uk.gov.hmcts.reform.draftstore.features.store;
+
+import com.google.common.collect.ImmutableMap;
+
+import org.json.JSONException;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.context.junit4.SpringRunner;
+import uk.gov.hmcts.reform.draftstore.FunctionalTestConfiguration;
+import uk.gov.hmcts.reform.draftstore.steps.DraftStoreSteps;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.springframework.http.HttpStatus.OK;
+
+@TestPropertySource("classpath:application.properties")
+@RunWith(SpringRunner.class)
+@ContextConfiguration(classes = FunctionalTestConfiguration.class)
+public class CreateDraftTest {
+
+    @Autowired
+    private DraftStoreSteps draftStoreSteps;
+
+    @Test
+    public void user_successfully_stores_new_draft() throws JSONException {
```

Comment:
let's stick do describing behaviour, not end result

Reply:
This name would be good for smoke tests, which are the happy scenario tests (i.e. check if we can do this). Functional tests have various edge cases and therefore should follow something like this:
```
public void should-{what we expect it do to}-when-{conditions}() { ...
```